plugins/lsp/lspsaga.nix

```diff
@@ -201,19 +205,19 @@ in
       };
 
       finder = {
-        maxHeight = helpers.defaultNullOpts.mkNullable (types.numbers.between 0.0 1.0) "0.5" ''
+        maxHeight = helpers.defaultNullOpts.mkNullable (types.numbers.between 0.0 1.0) 0.5 ''
           `max_height` of the finder window (float layout).
         '';
 
-        leftWidth = helpers.defaultNullOpts.mkNullable (types.numbers.between 0.0 1.0) "0.3" ''
+        leftWidth = helpers.defaultNullOpts.mkNullable (types.numbers.between 0.0 1.0) 0.3 ''
           Width of the left finder window (float layout).
         '';
 
-        rightWidth = helpers.defaultNullOpts.mkNullable (types.numbers.between 0.0 1.0) "0.3" ''
+        rightWidth = helpers.defaultNullOpts.mkNullable (types.numbers.between 0.0 1.0) 0.3 ''
```

Comment:
Maybe we should add a `mkNumber01` helper at some point...

Reply:
And/or `mkBetween`?